### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by bleonard**
overall, jsut trying to remove notes of resque
- readme
- rake files to what's actually being used

### CODE REVIEW

File: README.mdown [hunk 1]
```diff
@@ -110,122 +112,11 @@ Note: This subscribes when this class is loaded, so it needs to be in your load
...
-  end
-
-  # runs at 8am on the first of every month
-  subscribe "new_month_morning", 'bus_event_type' => 'heartbeat_minutes', 'day' => 1, hour' => 8, 'minute' => 0,  do |attributes|
-    next unless attributes["epoch_minutes"] % 5 == 0
-    Token.old.expire!
-  end
-end
-```
+    $ bundle exec sidekiq
```
Comment (bleonard) [inline README.mdown]: @jonsgreen do we need to pass sidekiq the queues to go after or does this discover them?

Comment (bleonard) [inline README.mdown]: Ah. Found this: https://github.com/mperham/sidekiq/wiki/Advanced-Options

Comment (bleonard) [inline README.mdown]: Here's a way to list queues for reference. adding to queue-bus rake jos

``` ruby
  desc "List QueueBus queues that need worked"
  task :queues => [ :preload ] do
    manager = ResqueBus::TaskManager.new(false)
    queues = manager.queue_names + ['bus_incoming']
    puts queues.join(", ")
  end
```

**Comment by bleonard** [comment]
@jonsgreen I forgot that i"d also cleaned up the rake files a while back. I merged that in and these readme changes and bumped the version for you. push to rubygems. please check it out.

**Comment by jonsgreen** [comment]
Just want to let you know that this is running on production and so far all is well though traffic is still light.

We did need a custom wrapper for setting a separate Sidekiq ConnectionPool in order to use an external Redis Connection but fortunately Sidekiq 3.0 makes this easy to do. 

Thanks again! Time to let Ruby5 know.

**Comment by bleonard** [comment]
yeah! i'll be doing some worth this month on the resque version and getting us upgraded. then we'll spread the word!

**Comment by bleonard** [comment]
@jonsgreen I wrote a blog post! http://tech.taskrabbit.com/blog/2015/04/02/queue-bus/
Feel free to share on ruby5, twitter, etc.

I did these ones:
https://news.ycombinator.com/newest
http://www.rubyflow.com/p/hx76sm-queue-bus

**Pull request merged**